tests/ui/agents/test_agents_viaf_tasks.py

```
@@ -0,0 +1,265 @@
+    mock_get.return_value = mock_response(json_data=agent_viaf_online_response)
+
+    # Refresh (create) the record
+    action = _refresh_viaf_record(pid=pid, dbcommit=True, reindex=True, verbose=True)
+
+    assert action == Action.CREATE
+    assert AgentViafRecord.get_record_by_pid(pid) is not None
+
+
+@mock.patch("requests.Session.get")
+def test_refresh_viaf_record_update(
+    mock_get, app, agent_viaf_record, agent_viaf_online_response
```

Ruff (0.15.2)

[warning] Unused function argument: `app`

(ARG001)